telehealth-example/src/main/java/com/pubnub/components/example/telehealth/viewmodel/LoginViewModel.kt

```diff
@@ -0,0 +1,23 @@
+package com.pubnub.components.example.telehealth.viewmodel
+
+import androidx.lifecycle.ViewModel
+import com.pubnub.components.data.member.DBMemberWithChannels
+import com.pubnub.components.example.telehealth.ChatApplication
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+
+class LoginViewModel : ViewModel() {
+
+    lateinit var members : List<DBMemberWithChannels>
+
+    init {
+        getMembersFromDb()
```

Comment:
Since the code is called asynchronously, we cannot guarantee that the `members` will contains a list of objects. I'll rather use it to check if a member with provided username exists in the database (as a suspend function)